fix/security vulnerability in package

dasarathg68 opened the pull request:
# Description

Fixes #80 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines

hermannleboss commented:
How you solve this ?

I tried to upgrade packages and the project vulnerability appear.

dasarathg68 commented:
> How you solve this ?
> 
> I tried to upgrade packages and the project vulnerability appear.

@hermannleboss I didn't know before that you were trying to upgrade packages (wasn't mentioned in the issue).
I have fixed it now, fixed check. 
Changes made:
redoc-cli is no longer being maintained so we migrated from redoc-cli to redocly.

hermannleboss reviewed (approved):
LGTM
